code/handlers/trackservers.q

```diff
@@ -333,13 +333,23 @@ reqprocsnotconn:{[requiredprocs]
     not all requiredprocs in exec proctype from .servers.SERVERS where .dotz.liveh[w]
   }
 
+startupdependent:{ 
+     n:();                                                                                          //variable used to check how many cycles have passed
```

Comment:
The indentation in this file is all over the place (I counted 12 spaces on line 314) but I think you should use 4 spaces indentation throughout this function because (a) it's used elsewhere here and (b) it matches the large indentation anyway.

Also `n` should be initialised as `0`.